pennylane_lightning/core/src/simulators/lightning_qubit/gates/tests/Test_KernelMap.cpp

```diff
@@ -35,7 +35,7 @@ using Pennylane::Util::LightningException;
 
 TEST_CASE("Test PriorityDispatchSet", "[PriorityDispatchSet]") {
     auto pds = PriorityDispatchSet();
-    pds.emplace(Pennylane::Gates::KernelType::PI, 10U,
+    pds.emplace(Pennylane::Gates::KernelType::LM, 10U,
```

Comment:
I guess we can just remove it

Reply:
Let's keep it to ensure that the priority dispatcher always works as expected even on Windows with one single gate match.